wp/lib/bap_wp/src/precondition.ml

```diff
@@ -941,17 +913,12 @@ let visit_graph (env : Env.t) (post : Constr.t)
         let handler = Env.get_loop_handler env in
         post, handler env post ~start:dst g
       end
-    | _ -> p
+    | _ -> post, env
```

Comment:
again, nice catch. Maybe worth leaving a comment here about why this `post` is the good `post`.

As a sign `post` for other confused readers.

Reply:
👍